Core.ApplicationServices/Contract/ItContractService.cs

```diff
@@ -94,6 +94,7 @@ public IQueryable<ItContract> GetAllByOrganization(int orgId, string optionalNam
                         return deleteByContractId.Error;
                     }
                     _domainEvents.Raise(new ContractDeleted(contract));
+                    _domainEvents.Raise(new EntityDeletedEvent<ItContract>(contract)); // Don't need the contract itself for the ItSystemUsageOverviewReadModel updates.
```

Comment:
kommentaren forstår jeg ikke... :-) Den smider også et domain event ovenfor. Kan vi ikke bare abonnere på den i stedet for at subscribe til endnu en? Det er jo 0..* på handlers